### ISSUE TITLE
PRNG is not seeded

### ISSUE THREAD

**Issue by shadowfox87** [termux/proot#129]
After the most recent update for proot, ssh has stopped working. If I run `ssh` , it will say "PRNG is not seeded". If I run outside of linux, ssh works just fine. I've tried this on Ubuntu 20.04 and also the latest version of archlinux.

See the below screenshot.
https://cdn.discordapp.com/attachments/411604294723305483/772183392568737812/unknown.png

**Comment by HKunogi** [comment]
Happens to me on ubuntu and on archlinux.
On Kali Nethunter just after installing it works fine, but after update && upgrade, the same problem appears there.

**Comment by michalbednarski** [comment]
Doesn't happen on my test devices, could you share what you've used to install distro, Android version and results of following commands

```
uname -a
apt list proot
```

**Comment by HKunogi** [comment]
I've tested into an ARMv81 and on an i686 device, and the issue seems to be that the entropy is too low, would be good if we could have an fake entropy generator just for having stuff like openssl that requires certain entropy level to work (as an option, even if its not secure)

**Comment by shadowfox87** [comment]
Yes, I can confirm that downgrading the openssl version fixed the issue. You should test it on a pc not mobile.
